Week2/Week 2 Solution(2).txt

```diff
@@ -0,0 +1,85 @@
+C:\sqlite3>sqlite3
+SQLite version 3.36.0 2021-06-18 18:36:39
+Enter ".help" for usage hints.
+Connected to a transient in-memory database.
+Use ".open FILENAME" to reopen on a persistent database.
+sqlite> create table stud(id int primary key, name text);
+sqlite> insert into stud values(1,'om');
+sqlite> insert into stud values(2,'sai');
+sqlite> insert into stud values(3,'ram');
+sqlite> insert into stud values(4,'Radha');
+sqlite> insert into stud values(5,'Krishna'),
+			       (6,'Krish'),
+                               (7,'Kusum'),
+                               (8,'Anamika'),
+                               (9,'Pooja'),
+                               (10,'Amit');
+sqlite> select * from stud;
+1|om
+2|sai
+3|ram
+4|Radha
+5|Krishna
+6|Krish
+7|Kusum
+8|Anamika
+9|Pooja
+10|Amit
+
+************************************************************************************************************
+
+sqlite> create table stud_details
+   ...> (
+   ...> id int REFERENCES stud(id),
+   ...> city text,
+   ...> mobile no int
+   ...> );
+sqlite> insert into stud_details values(101,'Surat',1234567890),
+   ...> (102,'Bombay',9876543210),
+   ...> (103,'Bengaluru',2345678910),
+   ...> (104,'Delhi',1013456789),
+   ...> (105,'Mumbai',3456789021),
+   ...> (106,'kolkata',4561567450),
+   ...> (107,'Gujarat',6354184566),
+   ...> (108,'Vapi',9662345678),
+   ...> (109,'Ahmedabad',678905432),
+   ...> (110,'navsari',3456271904);
+sqlite> select * from stud_details;
+101|Surat|1234567890
+102|Bombay|9876543210
+103|Bengaluru|2345678910
+104|Delhi|1013456789
+105|Mumbai|3456789021
+106|kolkata|4561567450
+107|Gujarat|6354184566
+108|Vapi|9662345678
+109|Ahmedabad|678905432
+110|navsari|3456271904
+ create table stud
+   ...> (
+   ...> id int primary key,
+   ...> name text
+   ...> );
+sqlite> insert into stud values(101,'om'),
+   ...> (102,'Sai'),
+   ...> (103,'Ram'),
+   ...> (104,'Radha'),
+   ...> (105,'Krishna'),
+   ...> (106,'Krish'),
+   ...> (107,'Kusum'),
+   ...> (108,'Pooja'),
+   ...> (109,'anamika'),
+   ...> (110,'Amit');
+sqlite> select * from stud;

```

Comment:
if i want to print students id between 101 to 105, which query i will have to fire ?

Reply:
select * from stud where stud_id BETWEEN 101 AND 105;